README.md

```diff
@@ -1,3 +1,22 @@
 # The flatpak manifest for Bitwarden Desktop
 
 Please point issues with the flatpak release to https://github.com/bitwarden/clients/issues.
+
+## Experimental Wayland support
+
+Electron (the technology used to build the Bitwarden Desktop app) may not play nicely with Wayland at the current moment, especially in certain desktop environments.
```

Comment:
```suggestion
Electron (the technology used to build the Bitwarden Desktop app) may not play nicely with Wayland at the current moment, especially in GNOME.
```

Reply:
Is there any other DE known to cause issues?

Reply:
I'm not sure, that's why I used that wording, it works fine in KDE, but I don't know any other DEs.